src/web-auth/web-message-handler.js

```diff
@@ -15,7 +15,10 @@ function runWebMessageFlow(authorizeUrl, options, callback) {
       }
     },
     timeoutCallback: function() {
-      callback({ error: 'timeout', error_description: 'Timeout during fetching SSO data' });
+      callback({
+        error: 'timeout',
+        error_description: 'Timeout during executing web_message communication'
```

Comment:
isn't this a breaking change? I don't see it documented for the users.

Reply:
error_description is not meant to be parsed by anyone. I just made it more generic, since we're now using web_message for different things.
In any case, the v9 release is a new major, so that's ok.

Reply:
I'm asking because in some sdks we rely on this "error_description" field to parse (wrong designed) error responses. I'm ok with this change as long as it's documented somewhere soon. i.e. add a file describing the possible configuration/runtime errors.

Reply:
cool. we'll add to the migration guide.